Cargo.toml

```diff
@@ -26,7 +26,7 @@ num-traits = "0.2"
 reqwest = { version = "0.12", default-features = false, features = [
   "rustls-tls-native-roots",
 ] }
-review-database = { git = "https://github.com/petabi/review-database.git", tag = "0.29.1" }
+review-database = { git = "https://github.com/petabi/review-database.git", rev = "0340144" }
```

Comment:
It would be better to use `66ccdd49` because it includes the clarified README.

Reply:
I replaced the commit hash `0340144` with `66ccdd49`.